app/controllers/api/v2/courses_controller.rb

```diff
@@ -22,33 +22,37 @@ def show
       end
 
       def sync_with_search_and_compare
-        response = sync_courses
-
-        if response
-          head :ok
-        else
-          raise RuntimeError.new(
-            'error received when syncing with search and compare'
+        if @course.syncable?
+          response = SearchAndCompareAPIService::Request.sync(
+            [@course]
           )
+          if response
```

Comment:
reduce unneeded `if value`? to `if !value`

Reply:
hold on did you rebase tis?

Reply:
Done and have now rebased 👍